packages/traceability-schemas/scripts/schemas-to-vocab.js

```diff
@@ -129,9 +121,49 @@ const separateSchemas = (schemaList) => {
   return { credentials, vocab };
 };
 
+const buildWorkflowSection = () => {
+  const images = {
+    "Business Card Workflow": "resources/wf_businesscard.svg"
```

Comment:
Having to add the image reference here seems wrong. The workflow yml should be self-contained. 

How about adding the SVG as an element alongside mermaid. 

```
mermaid: >-
  sequenceDiagram
      participant A as Alice
      ...
svg: >-
  <svg aria-roledescription="sequence"
  ...
    
```